src/context.rs

```diff
@@ -1216,6 +1216,19 @@ impl SeafowlContext for DefaultSeafowlContext {
                         })),
                     }))
                 }
+                Statement::DropFunction{
+                    if_exists,
+                    func_desc,
+                    option: _
+                } => {
+                        Ok(LogicalPlan::Extension(Extension {
+                            node: Arc::new(SeafowlExtensionNode::DropFunction(DropFunction {
+                                if_exists,
+                                func_desc,
```

Comment:
You could also extract those function names all the way up here (and change the corresponding field in `SeafowlExtensionNode::DropFunction` to be `func_names: Vec<String>`).

Reply:
Done 👍